### PR TITLE
Art Cortez

### PR DESCRIPTION
**Pull request by orctez**
# Checkpoint Rubric

This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

## Checkpoint 1: Company/Organization Website following Wireframe
- [ ] A CSS file is effectively linked to the HTML file in the Head section: 10 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 10 points 
- [ ] The webpage links to another page (multi-page website): 10 points
- [ ] The company/orgnanization name is lined up with the navigation menu at the top of the page: 10 points
- [ ] The title is centered in front of a background image: 10 points
- [ ] The body images are sized and formatted according to the wire frame: 10 points
- [ ] The body text is sized, positioned, and formatted to match the fire frame: 10 points
- [ ] Flexbox or floats are used to accurately replicate the layout of the wireframe: 10 points
- [ ] The footer consists of a background image, centered text, and 2 buttons that function as links: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 2: Foundry Winery Website
- [ ] Responsive navbar: 20 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 30 points 
- [ ] Matched layout of original site: 20 points
- [ ] Matched font of original site: 10 points
- [ ] Properly commented code: 10 points
- [ ] Properly indented code: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 3: Learning Team Project
- [ ] Mockup meets customer requirements: 25 points
- [ ] Website meets customer requirements: 25 points
- [ ] Learning team worked together effectively and contributed equally: 25 points
- [ ] Class presentation was engaging and effective: 25 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 4: Personal Portfolio Website
- [ ] Responive: 20 points
- [ ] Evidence of Effective Use of Design Principles: 20 points
- [ ] Links to Resume: 20 points
- [ ] Links to at least 3 Projects: 20 points
- [ ] Effective Communicates Desired Message ("Hire me"): 20 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

### CODE REVIEW

**Comment by matthewMurray14** [comment]
looks like you might have some indenting errors.

File: index.html [hunk 1]
```diff
@@ -1,2 +1,12 @@
-<!DOCTYPE html>
-<h1>Hello World</h1>
```
Comment (stefanbemelmans) [inline index.html]: Needs to be within the <html> tag

File: 02week/nasa/css/style.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    text-align: center;
+    font-size: 100px;
+    color: white;
+    padding: 150px;
+
+
+}
+
+.nasa {
+    background-image: url(/Users/orctez/Desktop/github/web-workbook/02week/nasa/img/nasa-background.jpg);
```
Comment (kevincolten) [inline 02week/nasa/css/style.css]: Make sure this is a relative link. `'../img/nasa-background.jpg'`

**Review by orctez — COMMENTED** [review]
?

**Pull request closed without merging**